tidb-cloud/data-service-manage-endpoint.md

```diff
@@ -238,6 +241,14 @@ TiDB Cloud generates code examples to help you call an endpoint. To get the code
 5. Paste the code example in your application and run it.
 
     - You need to replace the `<Public Key>` and `<Private Key>` placeholders with your API key. For more information, refer to [Manage an API key](/tidb-cloud/data-service-api-key.md).
+    - To paginate the results for a `GET` request, you can replace `<Page Number>` and `<Page Size>` with actual values. For example, to get the second page with 10 items per page, use `api_page=2` and `api_page_size=10`.
+
+        > **Note:**
+        >
+        > - The pagination feature is available only for `GET` requests where the last query is a `SELECT` statement.
+        > - If you do not include the `api_page` and `api_page_size` parameters, the default behavior is to return the first page with the maximum number of rows specified in the **Max Rows** property.
+        > - The `api_page * api_page_size` must be less than the **Max Rows** property. Otherwise, an error is returned.
```

Comment:
```suggestion
        > - The `api_page_size` must be less than or equal to the **Max Rows** property. Otherwise, an error is returned.
```
@jusong PTAL